micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/EhCache2Metrics.java

```diff
@@ -102,7 +103,7 @@ protected void bindImplementationSpecificMetrics(MeterRegistry registry) {
                 .description("Cache puts resulting in a new key/value pair")
                 .register(registry);
 
-        FunctionCounter.builder("cache.puts.added", stats, StatisticsGateway::cachePutAddedCount)
+        FunctionCounter.builder("cache.puts.added", stats, StatisticsGateway::cachePutUpdatedCount)
```

Comment:
I wonder now if this meter is appropriately named. Perhaps it should be just `cache.puts`, keeping the result tag that indicates the action verb.

Reply:
Good point! I've changed meter name to `cache.puts.updated` and removed qualifying tags since names are unique now.
IMO it's better to have fully qualified names here rather than `cache.puts` since we already have this name defined in https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/CacheMeterBinder.java#L77
which corresponds to total number of puts per cache not segregated by added/modified flavor.

Reply:
Ah, I see. So it was a name disambiguation concern originally. I don't find `updated` to be any more accurate than `added`, so perhaps we should revert to `added` to preserve compatibility?

Reply:
Done